app/src/main/java/com/example/mylittlelibrary/dao/LibraryApi.kt

```diff
@@ -1,4 +1,4 @@
-package com.example.mylittlelibrary.service.api
+package com.example.mylittlelibrary.dao
```

Comment:
Added dao package to follow the MVVM best practices.
https://blog.rsuter.com/recommendations-best-practices-implementing-mvvm-xaml-net-applications/#:~:text=All%20views%20should%20follow%20the,has%20exactly%20one%20view%20model.